docs/overview/office-add-in-availability.md

```diff
@@ -396,9 +382,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
     <td> - TaskPane<br>
          - <a href="https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/add-in-commands-requirement-sets">Add-in Commands</a></td>
```

Comment:
Need to delete this line (383) because we don't support Add-in Commands in Office 2016 for Windows (except in Outlook).